internal/rbd/rbd_util.go

```diff
@@ -1063,16 +1054,14 @@ func cleanupRBDImageMetadataStash(path string) error {
 
 // resizeRBDImage resizes the given volume to new size.
 func resizeRBDImage(rbdVol *rbdVolume, cr *util.Credentials) error {
-	var output []byte
-
 	mon := rbdVol.Monitors
 	volSzMiB := fmt.Sprintf("%dM", util.RoundOffVolSize(rbdVol.VolSize))
 
 	args := []string{"resize", rbdVol.String(), "--size", volSzMiB, "--id", cr.ID, "-m", mon, "--keyfile=" + cr.KeyFile}
-	output, err := execCommand("rbd", args)
+	_, stderr, err := util.ExecCommand(context.TODO(), "rbd", args...)
```

Comment:
same here also

Reply:
this one is straight forward, I'll add that